portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,12 +34,17 @@
 @WebServlet("/data")
```

Comment:
@WebServlet("/comments") would be a better name for the API since it's mainly for manipulating the comments data.